zanata-war/src/main/resources/messages.properties

```diff
@@ -1231,10 +1249,10 @@ jsf.AlreadyInGroup=Already in Group
 jsf.email.joingrouprequest.AdditionalInfoMessage=To ensure your request is processed without delay, please provide any additional information that will help the group maintainers to process your request.
 !FIXME make key similar to related keys
 !{0} = locale id. e.g. en-us, {1} language display name. e.g. English
-jsf.email.RequestToJoinLanguageTeam.title=Request to join {0} [{1}] language team
+jsf.email.RequestToJoinLanguageTeam.title=Request to join {0} language team
```

Comment:
- [ ] SF: Did we really mean to remove the {1} language display name? It's still listed in the comment above.